.github/workflows/ci-cd.yaml

```diff
@@ -53,20 +53,28 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v3
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+
       - name: Setup Node.js
         uses: actions/setup-node@v3
         with:
           node-version: 16
-      - run: npm ci
-      - name: Release
+
+      - name: Install Dependencies
+        run: npm ci
+
+      - name: Create Release Pull Request or Publish to npm
+        id: changesets
+        uses: changesets/action@v1
+        with:
+          version: npm run version
+          publish: npm run release
+          commit: "chore(changesets): version packages"
+          title: "chore(changesets): version packages"
+          createGithubReleases: false
```

Comment:
I think we will want this `true` since semantic-release was the one creating these for us.

Reply:
fixed